web/source/kmwosk.js

```diff
@@ -2801,6 +2800,11 @@
     
     for(nLayer=0;nLayer<layers.length; nLayer++)
     {
+      // Check the heights of each row, in case different layers have different row counts.
+      nRows=layers[nLayer].childNodes.length;
+      rowHeight=Math.floor(oskHeight/nRows);
```

Comment:
Is it ever possible for layers to not have childNodes and `oskHeight/nRows` becomes a divide by 0?